js/application.js

```diff
@@ -176,17 +192,18 @@ var findBall = function (ballLocatedAt) {
 //------------------------------------------------------------
 function Game() {
   //variables
-  this.cupArray = [];
-  this.movesArray = [];
+  var cupArray = [];
+  var movesArray = [];
   this.numberOfCups = 3;
-  this.startPosition;
+  var playerMoveArray = [];
+  var startPosition;
 
```

Comment:
Please look at the example at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#Using_a_constructor_function.

Create variables inside the game with `this.cupArray = []` instead of `var cupArray = []`